com.unity.render-pipelines.high-definition/Runtime/Lighting/ScreenSpaceLighting/BilateralUpsample.compute

```diff
@@ -10,78 +9,103 @@
 #include "Packages/com.unity.render-pipelines.high-definition/Runtime/ShaderLibrary/ShaderVariables.hlsl"
 #include "Packages/com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Raytracing/Shaders/RayTracingCommon.hlsl"
 #include "Packages/com.unity.render-pipelines.high-definition/Runtime/Lighting/ScreenSpaceLighting/BilateralUpsample.hlsl"
+#include "Packages/com.unity.render-pipelines.high-definition/Runtime/Lighting/ScreenSpaceLighting/BilateralUpsampleDef.cs.hlsl"
 
 // Mip chain depth buffer
 TEXTURE2D_X(_DepthTexture);
 // The half resolution texture that needs to be upscaled
 TEXTURE2D_X(_LowResolutionTexture);
 
-// Constant buffer where all variables should land
-CBUFFER_START(UnityScreenSpaceGlobalIllumination)
-    float4 _HalfScreenSize;
-    float2 _DepthPyramidFirstMipLevelOffset;
-CBUFFER_END
+// LDS that store the half resolution data
+groupshared float3 gs_cacheLighting[36];
```

Comment:
Compressing the result to 11 11 10 makes the kernel overall slower unfortunately.